src/app/inbound/mod.rs

```diff
@@ -123,7 +130,10 @@ pub fn spawn<P>(
             profiles_client,
             dst_route_layer,
         ))
-        .push(strip_header::request::layer(super::DST_OVERRIDE_HEADER));
+        .push(strip_header::request::layer(super::DST_OVERRIDE_HEADER))
+        .push(trace::layer(
+            |dst: &DstAddr| info_span!("logical", dst=%dst.dst_logical()),
```

Comment:
Nit/style: prefer spaces around `=`
```suggestion
            |dst: &DstAddr| info_span!("logical", dst = %dst.dst_logical()),
```